app/models/user.py

```diff
@@ -78,4 +78,9 @@ def isProgramManagerFor(self, program):
     def isProgramManagerForEvent(self, event):
         # Looks to see who the Program Manager for a specific event is
         return self.isProgramManagerFor(event.program)
+    
+    def isProgramManager(self):
```

Comment:
Let's make this a property rather than a method, because in usage it is just like `isCeltsAdmin` or `isAdmin`